### ISSUE TITLE
OpenSSH doesn't work

### ISSUE THREAD

**Issue by rweichler** [Sticktron/g0blin#242]
I'm pretty sure you just need to use dropbear. Here's what happens when I manually try to start sshd:

```
key load public: invalid format
Could not load host key: /etc/ssh/ssh_host_key
Could not load host key: /etc/ssh/ssh_host_rsa_key
Could not load host key: /etc/ssh/ssh_host_dsa_key
Disabling protocol version 2. Could not load host key
sshd: no hostkeys available -- exiting.
```

**Comment by nik312123** [comment]
@rweichler I agree! I use Dropbear from ichitaso's beta repository. If you use this Dropbear, though, you must be sure to use port 2222.

e.g.
`ssh -p 2222 root@ipAddressFromWiFi`

**Comment by cgarst** [comment]
Got OpenSSH working by doing the following in MTerminal:
`su`
`ssh-keygen -f /etc/ssh/ssh_host_rsa_key -N '' -t rsa`
`ssh-keygen -f /etc/ssh/ssh_host_dsa_key -N '' -t dsa`
`reboot`

**Comment by resistor4u** [comment]
@cgarst can you share the output of `find / -iname "ssh*"` run as root from the phone?
